usagov/vote-gov-drupal
# Bug/vote 1151/add lang name translations

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1151

## Description 
Added Chinese native language names
Created a new language link title value with both language names (preprocess.inc) 

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. Run lando retune

### Testing steps
1. Verify that each language link in the dropdown shows up as "[native language name] ([english lang name])" if the two names are different
2. Verify the Chinese native names display
3. Click on each language to load the translated pages, and repeat steps 1 and 2
